docs/setting-up/server/index.md

```diff
@@ -25,3 +25,19 @@ Choose how you want to run PMM Server:
 - [on an Amazon AWS EC2 instance](aws.md)
 
 When PMM Server is running, set up [PMM Client](../client/index.md) for each node or service.
+
+## Single line installer
+
+> <b style="color:goldenrod">Caution</b> This is a [technical preview](../../details/glossary.md#technical-preview) and is subject to change.
+
+```sh
+curl -fsSL https://raw.githubusercontent.com/percona/pmm/PMM-2.0/get-pmm.sh -o get-pmm2.sh ; chmod +x get-pmm2.sh ; ./get-pmm2.sh
```

Comment:
Is this doing the same thing that caused chaos with codecov (wget <filename> | sh?). I realize we're not piping directly to shell but should we encourage users to pause, checksum/validate/look at the code before we have them execute a shell script they downloaded?

Reply:
There should be an admonition to that effect, right under the code block.